tower-spawn-ready/src/make.rs

```diff
@@ -11,8 +17,10 @@ pub struct MakeSpawnReady<S, E = DefaultExecutor> {
 }
 
 /// Builds a SpawnReady with the result of an inner Future.
+#[pin_project]
```

Comment:
we should prob just drop the `E` since all the executor traits are in flux anyways.